ecs-cli/modules/compose/ecs/service.go

```diff
@@ -68,24 +69,33 @@ func (s *Service) LoadContext() error {
 		MinimumHealthyPercent: minHealthyPercent,
 	}
 
+	// Load Balancer
+	role := s.Context().CLIContext.String(RoleFlag)
 	targetGroupArn := s.Context().CLIContext.String(TargetGroupArnFlag)
+	loadBalancerName := s.Context().CLIContext.String(LoadBalancerNameFlag)
 	containerName := s.Context().CLIContext.String(ContainerNameFlag)
 	containerPort, err := getInt64FromCLIContext(s.Context(), ContainerPortFlag)
 	if err != nil {
 		return err
 	}
-	loadBalancerName := s.Context().CLIContext.String(LoadBalancerNameFlag)
 
-	s.loadBalancer = &ecs.LoadBalancer{
-		ContainerName: aws.String(containerName),
-		ContainerPort: containerPort,
-	}
-
-	// AWS API server side validation will inform the user about not specifying both.
-	s.loadBalancer.TargetGroupArn = aws.String(targetGroupArn)
-	s.loadBalancer.LoadBalancerName = aws.String(loadBalancerName)
+	if role != "" || targetGroupArn != "" || loadBalancerName != "" || containerName != "" || containerPort != nil {
```

Comment:
Please add a comment here with a similar explanation as to what you have in the commit message. Also, can you add/update unit test(s) for this?